eng/Versions.props

```diff
@@ -65,7 +65,7 @@
     <MicrosoftNETCoreAppRefVersion>6.0.0-preview.4.21253.7</MicrosoftNETCoreAppRefVersion>
     <MicrosoftVisualBasicVersion>10.1.0</MicrosoftVisualBasicVersion>
     <MicrosoftVisualStudioCompositionVersion>16.1.8</MicrosoftVisualStudioCompositionVersion>
-    <NewtonsoftJsonVersion>12.0.1</NewtonsoftJsonVersion>
+    <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
```

Comment:
❓  This is only referenced by **PerfDiff.csproj**. If that project is going to be excluded from source build anyway, why do we need to update this property?

Reply:
Thanks.  I didn't realize that.  I reverted the newtonsoft.json change.